lego/apps/flatpages/serializers.py

```diff
@@ -10,10 +13,22 @@ class Meta:
         fields = ("pk", "title", "slug", "category")
 
 
-class PageDetailSerializer(BasisModelSerializer):
+class PageDetailSerializer(BasisModelSerializer, ObjectPermissionsSerializerMixin):
     content = ContentSerializerField()
     picture = ImageField(required=False, options={"height": 500})
 
     class Meta:
         model = Page
         fields = ("pk", "title", "slug", "content", "picture", "category")
+
+
+class PageDetailAuthSerializer(BasisModelSerializer, ObjectPermissionsSerializerMixin):
```

Comment:
Imo. this is a good choice, since it aligns with other serializer. We don't have a strict naming scheme, and the name explains the use, since it is for PageDetail when a user is authed.